plugin/karma.c

```diff
@@ -228,6 +251,8 @@ static void karma_handler(cbot_event_t event, cbot_actions_t actions)
       actions.send(event.bot, event.channel, "sorry, you're not authorized to do that!");
     }
     recapfree(captures);
+  } else if (reexec(forget, event.message, NULL) != -1) {
```

Comment:
Maybe I should also rename `increment` to something more descriptive, like `addressed`?